src/openrct2/peep/Peep.h

```diff
@@ -52,13 +52,15 @@ struct Ride;
 class GameActionResult;
 using ParkEntranceIndex = uint8_t;
 
-enum class PeepType : uint8_t
+namespace Crowd
 {
-    Guest,
-    Staff,
-
-    Invalid = 0xFF
-};
+    enum class Type : uint8_t
+    {
+        Guest,
+        Staff,
+        Invalid = 0xFF
+    };
```

Comment:
Sorry for the rework, we discussed a bit and I think the enums would suit well enough inside the `Peep` class themselves. Then we don't have to introduce a new namespace that is a synonym and it would look like:

```c++
struct Peep : SpriteBase
{
    enum class Type : uint8_t
    {
        ...
    };
    ...
}
```

And usage would be `Peep::Type::Staff`

Reply:
I agree that would be a better way to handle it.